.devcontainer/install.sh

```
@@ -0,0 +1,1088 @@
+# GitHub API call with optional authentication
+# Uses GITHUB_TOKEN if available (5000 req/h vs 60 req/h)
+github_api_call() {
+  local url="$1"
+  local auth_header=""
+
+  # Use token if available for higher rate limit
+  if [ -n "${GITHUB_TOKEN:-}" ]; then
+    auth_header="-H \"Authorization: token ${GITHUB_TOKEN}\""
+  fi
+
+  eval curl -sL $auth_header "$url" 2>/dev/null
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)